### ISSUE TITLE
[Adaptive Card iOS - Input.ChoiceSet]: Voice over focus is landing on the background elements while performing right swipe under 'What color do you want" combo box.'

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8427]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.ChoiceSet' and activate it.
7. Navigate to the 'What color do you want' combo box and double tap on it.
8. Now, perform the right swipe and observe the issue.
 
**Actual Result:**
After expanding the 'What color do you want' combo box, on performing the right swipe voice over focus is navigating to the background elements.

**Expected Result:**
On performing right swipe, voice over focus should be land on the menu items and it should announce the fist list item of the 'What color do you want'.

**Note:**
The issue is repro throughout application.

**User Experience:**
It will impact the screen reader user as they have to put more effort to navigate on the same control.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229742971-4adcfec0-9d2d-4ac7-bb28-6f168738f60a.mov

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
@vagpt 
Please note that any cards under the 'Test & Elements' section are intended solely for internal feature development. Once features have been verified, new cards are created and placed in the 'Scenario' section. It's important to focus testing efforts only on cards within the 'Scenario' section.

To address this, I'm currently working on fixing the issue so that cards from other folders will not appear in the sample app. Thank you for your understanding.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
